vmdb/app/helpers/ems_cloud_helper/textual_summary.rb

```diff
@@ -135,6 +135,21 @@ def textual_authentications
     end
   end
 
+  def textual_refresh_status
+    last_refresh_status =
+      if @ems.last_refresh_date
+        last_refresh_date = time_ago_in_words(@ems.last_refresh_date.in_time_zone(Time.zone)).titleize
+        "#{@ems.last_refresh_status.titleize} - #{last_refresh_date} Ago"
+      else
+        "Never"
```

Comment:
Since this is now in the refresh status,  maybe consolidate? 

```ruby
last_refresh_status = @ems.last_refresh_status.titleize
if @ems.last_refresh_date
  last_refresh_date = time_ago_in_words(@ems.last_refresh_date.in_time_zone(Time.zone)).titleize
  last_refresh_status << " - #{last_refresh_date} Ago"
end
```